ignite/metrics/running_average.py

```diff
@@ -87,6 +87,9 @@ def log_running_avg_metrics():
     """
 
     required_output_keys = None
+    # TODO Shall we put `src` here? Then we should add a new branch for metric-typed attributes in `state_dict`
+    # and `load_state_dict`. Examples; This class; `Rouge` which has a `List[_BaseRouge]`.
+    _state_dict_all_req_keys = ("_value",)
```

Comment:
@vfdev-5 , What's your thoughts on this?

Reply:
Let's add `src` and in state_dict/load_state_dict put an explicit check for `isinstance(attr, Metric)` and call `state[attr_name].update(attr.state_dict)` or something like that.
Do you have any other options ?